FileHandlingAssignment/userLogin.php

```diff
@@ -0,0 +1,32 @@
+<?php
+	include 'config.php';
+	
+	echo "In login PHP";
+	if(!$conn)
+	{
+		echo "Connection Failed";
+	}
+	else
+	{
+		$flag="false";
+		$userName=$_POST['userName'];
+		$password=$_POST['userPassword'];
+		$sqlQuery= "SELECT * FROM userDetails WHERE userName = '".$userName."' and password='".$password."'"; 
+		$statement = $conn->query($sqlQuery);
+		foreach($conn->query($sqlQuery) as $row)
```

Comment:
Why didn't you use the $statement variable when you've assigned the result of the query in that.
Also check if the data is not empty then only any of the loop should be called. 
Check and apply it everywhere